egs2/librispeech/asr1/local/data.sh

```diff
@@ -17,6 +17,7 @@ stop_stage=100000
 data_url=www.openslr.org/resources/12
```

Comment:
ditto
Where are you using this?
Is it required?

Reply:
ditto.